ui/src/app/event-sources/components/event-source-log-viewer.tsx

```diff
@@ -99,6 +100,7 @@ export const EventSourceLogsViewer = ({
                             }}
                         />
                     )}
+                    <Links scope='event-source-logs' object={eventSource} />
```

Comment:
Looks like event source logs were not actually added. We have this in the example configmap tho. So have to fix this.